### PR TITLE
tutorial: Music Jam

### PR DESCRIPTION
**Pull request by deadprogram**
This PR adds some of the music jam activities to the Badger2040.

Looks like it will needs a rebase after #5 is merged.

### CODE REVIEW

File: tutorial/musicjam/arpeggio-display/main.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+	// change this to speed up or slow down the music
+	bpm int = 80
+
+	currentNote  int = 0
+	currentNotes []midi.Note
+)
+
+func main() {
+	usb.Product = "TinyGo Arpeggio"
```
Comment (conejoninja) [inline tutorial/musicjam/arpeggio-display/main.go:48]: I really like the fact we can change the name of the USB device

**Comment by conejoninja** [comment]
This have the BLE tutorial and the music jam together. I'll wait until the BLE is removed from this OR changes for #5  are made.

**Comment by deadprogram** [comment]
OK corrected, the bluetooth and musicjam PRs have been separated.

**Comment by conejoninja** [comment]
Fantastic, thanks for your work! Merged

**Pull request merged**